src/framework/core/js/DataBinding.js

```diff
@@ -853,6 +853,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     };
 
     fluid.establishModelRelay = function (that, optionsModel, optionsML, optionsMR, applier) {
+        var shadow = fluid.shadowForComponent(that);
```

Comment:
Not sure what you mean "not more than once" - it does check that it is established any more times than once, and clearly it will be established *at least* once since this method has started to execute. The existing tests cases triggering this route are sufficient since numerous of them started to fail under the old framework when this check was introduced.